README.md

```diff
@@ -147,6 +147,13 @@ Summary
 
 ## Installation
 
+### Dependencies
+
+When installing with Cargo on Ubuntu you will need to have `libxrandr-dev` installed. To install it with apt run:
+```
+apt install libxrandr-dev
+```
```

Comment:
As you put this below as part of `From source`?

Reply:
@pickfire I just did it.